docs/internal-pipeline/archive_sync_service_messages.md

```diff
@@ -6,23 +6,23 @@ layout: default
 # Messages produced by archive-sync-service
 
 Archive Sync Service synchronizes every new archive by reading the related
-information from `ccx-XXX-insights-operator-archive-new` (where `XXX` needs to
-be replaced by environment, for example `prod`) Kafka topic, downloading the
-archive from AWS S3 and uploading it to DataHub (Ceph) bucket.
+information from `XXX-io-archive-new` (where `XXX` needs to be replaced by
```

Comment:
information from `[prod|qa]-io-archive-new`?

Reply:
I was keeping the same way of explaining it that was already in place. Do you want to substitute all the XXX by [qa|prod] regexp?